### ISSUE TITLE
pop3.py should conform to docstring and whitespace standards

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6666]
|[<img alt="staceysern's avatar" src="https://avatars.githubusercontent.com/u/3631162?s=50" width="50" height="50">](https://github.com/staceysern)| @staceysern reported|
|-|-|
|Trac ID|trac#6666|
|Type|enhancement|
|Created|2013-08-05 16:46:04Z|
|Branch|https://github.com/twisted/twisted/tree/pop3-doc-6666|

The docstrings in pop3.py should be updated to provide descriptions of all classes, functions and methods as well as their parameters and attributes. The spacing before classes and functions should conform to the coding standards.

<details><summary>Searchable metadata</summary>

```
trac-id__6666 6666
type__enhancement enhancement
reporter__shira shira
priority__normal normal
milestone__ 
branch__branches_pop3_doc_6666 branches/pop3-doc-6666
branch_author__shira shira
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__documentation documentation
time__1375721164000000 1375721164000000
changetime__1383246791000000 1383246791000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="staceysern's avatar" src="https://avatars.githubusercontent.com/u/3631162?s=50" width="50" height="50">](https://github.com/staceysern)<a name="note_1"></a>|@staceysern commented|
|-|-|

(In [39473]) Branching to 'pop3-doc-6666'

**Comment by twisted-trac** [comment]
|[<img alt="staceysern's avatar" src="https://avatars.githubusercontent.com/u/3631162?s=50" width="50" height="50">](https://github.com/staceysern)<a name="note_2"></a>|@staceysern commented|
|-|-|

(In [40042]) Revise docstrings and fix whitespace


refs [#6666](https://github.com/twisted/twisted/issues/6666)

**Comment by twisted-trac** [comment]
|[<img alt="staceysern's avatar" src="https://avatars.githubusercontent.com/u/3631162?s=50" width="50" height="50">](https://github.com/staceysern)<a name="note_3"></a>|@staceysern commented|
|-|-|

(In [40054]) Fixed twistedchecker errors

refs [#6666](https://github.com/twisted/twisted/issues/6666)

**Comment by twisted-trac** [comment]
|[<img alt="staceysern's avatar" src="https://avatars.githubusercontent.com/u/3631162?s=50" width="50" height="50">](https://github.com/staceysern)<a name="note_4"></a>|@staceysern removed owner|
|-|-|

This builds with twistedchecker error 9016 on line 1366 due to a bug in twistedchecker (bug report #1217379).

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)|@lvh set owner to @lvh|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_6"></a>|@lvh commented|
|-|-|

LGTM! Buildbot failure is there as you noted, but since that's a filed bug, I guess this can go in. I'll merge it for you :)

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_7"></a>|@lvh set status to `closed`|
|-|-|

(In [40546]) Merge pop3-doc-6666: Full API documentation for twisted.mail.pop3

Author: shira
Reviewer: lvh
Fixes: [#6666](https://github.com/twisted/twisted/issues/6666)

The POP3 support was previously missing a lot of API documentation. This adds full API documentation for twisted.mail.pop3.
